packages/@tinacms/cli/src/cmds/compile/index.ts

```diff
@@ -61,7 +61,8 @@ export const compile = async (_ctx, _next) => {
   }
 
   // Remove old js files
-  await resetGeneratedFolder
+  await fs.remove(tinaTempPath)
+  await fs.remove(tinaConfigPath)
```

Comment:
Take a look at this and make sure it is calling the most recent `removeGeneratedFolder` that you made in another PR: 
https://github.com/tinacms/tinacms/pull/2239